src/Google/GlobalSiteTag.php

```diff
@@ -355,6 +356,63 @@ public function maybe_display_conversion_and_purchase_event_snippets( string $ad
 			join( ',', $item_info ),
 		);
 		wp_print_inline_script_tag( $purchase_page_gtag );
+
+		// Enhanced conversion data.
+		$ec_data         = [];
```

Comment:
We should wrap all of this in an `if` statement that checks for the status of whether EC is active and only add this data when the status is 'active' to avoid unnecessary data. This should all be printed before the tracking events as well, so the user data is already set prior to the conversion or purchase events are fired.